src/stan_math_backend/Transform_Mir.ml

```diff
@@ -3,18 +3,49 @@ open Middle
 
 let use_opencl = ref false
 
-let opencl_triggers =
+let opencl_trigger_restrictions =
   String.Map.of_alist_exn
-    [ ( "normal_id_glm_lpdf"
-      , [ (* Array of conditions under which to move to OpenCL *)
-          [(1, UnsizedType.UMatrix)]
-        (* Argument 1 is a matrix *)
+    [ ( "bernoulli_lpmf"
+      , [ [ (0, UnsizedType.DataOnly, UnsizedType.UArray UnsizedType.UInt)
+          ; (1, UnsizedType.DataOnly, UnsizedType.UReal) ] ] )
```

Comment:
`(1, UnsizedType.DataOnly, UnsizedType.UReal)˙ means that we will not move to OpenCL if the second argument is of type real and is data or a parameter.